addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -68,6 +68,29 @@ MockServer.include({
         ];
         this._widget.call('bus_service', 'trigger', 'notification', [notifConfirmFold]);
     },
+    /**
+     * Simulate the 'channel_seen' route of of mail.channel
+     * In particular sends a notification on the bus
+     *
+     * @private
+     */
+     _mockChannelSeen(args) {
+        const { last_message_id } = args.kwargs;
+        const channel_id = args.args[0][0];
+        // Should use something of the form session.partner_id or env.messaging.currentPartner
+        // This is a limitation of the architecture of JS tests
+        const notification = [
+            ["dbName", 'res.partner', undefined],
+            {
+                info: 'channel_seen',
+                channel_id,
+                partner_id: undefined,
```

Comment:
This one is problematic though, there is no way this mock works without giving an id here.

Reply:
I know but there is no way to get that partner_id

Reply:
But we could with your PR

Reply:
what's the status of this PR ? merged soon ?

Reply:
I'm done with it, pending review  (4000+ 6000- diff...), depends on @alexkuhn

Reply:
I suppose it's #53826 ?

Reply:
> I suppose it's #53826 ?

yes, pending requested changes, depends on @seb-odoo 😝